pages/documentation/developer/scripting/python.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+---
+
+# Python scripting
+
+The [pypowsybl](../repositories/pypowsybl.md) provides a way to use all the features of PowSyBl in python script. This Python integration relies on the [JPype](https://www.py4j.org) project that provides full access to Java from Python. This is achieved not through re-implementing Python, as [Jython](https://www.jython.org/) has done, but rather through interfacing at the native level in both virtual machines.
+
+This shared memory based approach achieves decent computing performance, while providing the access to the entirety of CPython and Java libraries. For optimal performance, we recommend to use [Groovy scripts](groovy.md) or implement part of your script directly in Java.
+
+To make it easier to write Python scripts that use PowSyBL, we also provide a small DSL to:
+- to load and save a case file
+- to easily run power flow simulations with [OpenLoadFlow](../../simulation/powerflow/openlf.md)
+
+## Example
+This small example shows how to load a case file and run a power simulation in Python:
+```python
+import jpype
+import jpype.imports
+import pypowsybl
+
+# Start the JVM before importing java classes
+pypowsybl.start()
+
+from pypowsybl import Network, LoadFlow
+
+network = Network.load("eurostag-tutorial-example1.xml")
+result = LoadFlow.run(network, implementation="OpenLoadFlow")
+print("Computation OK? {}".format(result.isOk()))
+print("Metrics: {}".format(result.getMetrics()))
```

Comment:
And where is the save example?

Reply:
I didn't to give an example with all the features, but a quick overview of what you can do with pypowsybl. My plan is to write several tutorials instead. Do you think I should change the example to show how to export the grid?

Reply:
As you talk about 3 functions: load, run and save, I was expecting a save command line :-)

Reply:
Fine!